src/common/components/CardFaces.js

```diff
@@ -1,25 +1,39 @@
-export const cardFaces = [{
-    name: 'a',
-    face: '😛'
+import {
```

Comment:
For utils, I recommend importing from `common/utils` instead of exporting everything through `common`. As projects get larger, it's helpful to group components and utils separately.